components/filter/filter-dimension-set.js

```diff
@@ -132,6 +138,16 @@ class FilterDimensionSet extends LitElement {
 		return values;
 	}
 
+	willUpdate(changedProperties) {
+		if (changedProperties.has('hasMore') && this.hasMore) {
+			if (this.searchType !== 'manual') {
+				console.warn('Paging requires search type set to manual.');
+				this.hasMore = false;
+			}
+			else this.selectedFirst = true;
+		}
+	}
```

Comment:
Interested in some other opinions here.  Currently, if you've set `has-more` but not `search-type="manual"`, we turn off `has-more` and don't render the pager.  We can't just turn on manual search for them, because they won't have events setup to handle it. Then, if they _have_ correctly configured search, we override their setting of `selected-first` because we also require that to be on (and we can force that on for them, there's nothing they need to do).

This fails pretty gracefully, but just wondering if we should actually be more destructive to make it obvious things are misconfiguring?  Should we log a console error in one or both scenarios, and/or be less graceful and more obviously broken?

I'm good with this, but wanted to check since we've used a few different strategies across `core` before